### PR TITLE
some config Files

### PR DESCRIPTION
**Pull request by 928PJY**
add the config Files
@hellosnow @vwxyzh @chenkennt

### CODE REVIEW

File: src/VscPreviewExtension/.gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 node_modules/
-out/
\ No newline at end of file
+out/
+DfmParse/
+!.vscode/
```
Comment (hellosnow) [inline src/VscPreviewExtension/.gitignore]: why need to exclude .vscode folder especially?

File: src/VscPreviewExtension/README.md [hunk 2]
```diff
@@ -1,35 +1,120 @@
...
-* Bug fix: can't preview a single file(not in a open folder)
-* Bug fix: throw an error if it is not a markdown file
+# Document
+For further information and details about DocFX Flavored Markdown, please reference [DocFX Flavored Markdown](https://dotnet.github.io/docfx/spec/docfx_flavored_markdown.html)
 
-### 0.0.8
-* Bug fix: open other preview window
-* Bug fix: some css and javasript files to show the html
+# Feature Details
+## Live preview
```
Comment (hellosnow) [inline src/VscPreviewExtension/README.md]: >   [](start = 15, length = 2)

minor: remove extra spaces?

**Comment by hellosnow** [comment]
:shipit:

File: src/VscPreviewExtension/.gitignore (hunk 1 above)
Comment (928PJY) [inline src/VscPreviewExtension/.gitignore]: this folder is needed in the extension development

Comment (hellosnow) [inline src/VscPreviewExtension/.gitignore]: so I think it should be checked in? is it?

---
In reply to: [80604219](https://github.com/dotnet/docfx/pull/736#discussion_r80604219) [](ancestors = 80604219)

**Pull request merged**